modules/sdk-coin-sol/src/lib/tokenTransferBuilder.ts

```diff
@@ -64,12 +81,31 @@ export class TokenTransferBuilder extends TransactionBuilder {
     return this;
   }
 
+  /**
+   *
+   * @param {TokenAssociateRecipient} recipient - recipient of the associated token account creation
+   * @param {string} recipient.ownerAddress - owner of the associated token account
+   * @param {string} recipient.tokenName - name of the token that is intended to associate
+   * @returns {TransactionBuilder} This transaction builder
+   */
+  createAssociatedTokenAccount(recipient: TokenAssociateRecipient): this {
```

Comment:
do we need a separate function for this?

was thinking to update `send` to take a new param: `fundAtaCreation: boolean`.
when true, we would attach the create ATA instruction.